src/styles/breakpoints.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * xxl - Extra Extra Large
+ * xl - Extra Large
+ * lg - Large
+ * md - Medium
+ * sm - Small
+ * xs - Extra Small
+ */
+const breakpoints = {
+    xxl: '1400px',
+    xl: '1200px',
+    lg: '1025px',
+    md: '768px',
+    sm: '576px',
+    xs: '320px',
+};
+
+export const fontSize = {
```

Comment:
i like this approach but i think i would prefer this object to be in `styles/fonts.js`
what do you think?

Reply:
I believe that also works nicely, 
somewhere in my mind, I thought because this would be used across board on the application, it might want to be a file of its own.

Reply:
I reverted to having them in the breakpoint.ts file to avoid a cyclical dependency issue between font.ts and fontBreakpoint.ts

Reply:
@Darrellbor can you please explain to me what is cyclical dependency issue?

Reply:
Exporting from `fonts.ts` and importing into `fontsBreakpoint.ts` and vise versa